modules/calib3d/src/usac.hpp

```diff
@@ -0,0 +1,744 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_USAC_USAC_HPP
+#define OPENCV_USAC_USAC_HPP
+
+namespace cv { namespace usac {
+// Abstract Error class
+class Error : public Algorithm {
+public:
+    virtual void setModelParameters (const Mat &model) = 0;
+    virtual double getError (int point_idx) const = 0;
+};
+
+// Symmetric Reprojected Error
+class ReprojectedErrorSymmetric : public Error {
+public:
+    static Ptr<ReprojectedErrorSymmetric> create(const Mat &points);
+};
+
+// Forward Reprojected Error
+class ReprojectedErrorForward : public Error {
+public:
+    static Ptr<ReprojectedErrorForward> create(const Mat &points);
+};
+
+/*
+* Class for normalizing transformations of points.
+*/
+class NormTransform : public Algorithm {
+public:
+    virtual void getNormTransformation (Mat& norm_points, const std::vector<int>& sample,
```

Comment:
`Mat& norm_points` -> `Mat &norm_points` to have style consistency within your code

Reply:
fixed